docs/web-apps/automated-testing/electron.md

```diff
@@ -0,0 +1,72 @@
+---
+id: electron
+title: Electron Support on Sauce Labs [Beta]
+sidebar_label: Electron
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+[Electron](https://electronjs.org/) embeds Chromium and Node.js to enable web developers to create desktop applications. Sauce Labs is beta trialing support for automated testing with Electron-based apps on Windows 10 and 11, available to all users with VDC concurrency at US West or EU Central.
+
+## Supported Testing Platforms
+
+Sauce Labs currently supports the following test configurations for Electron.
+### Platforms
+Windows 10, Windows 11 (Linux and MacOS 13 support to come)
+### Electron versions
+versions 5 - 25
+
+## How to Get Started
+
+### App management
+
+Upload a zip file containing your Electron app via [REST API](/mobile-apps/app-storage/#uploading-apps-via-rest-api). You can refer to uploaded apps by either the app_id or app_name. Note that as with mobile apps, Electron apps are accessible only to members of the same team, and retained for 60 days.
```

Comment:
```suggestion
Upload a zip file containing your Electron app via [REST API](/mobile-apps/app-storage/#uploading-apps-via-rest-api). You can refer to uploaded apps by either the file ID or file name. Note that as with mobile apps, Electron apps are accessible only to members of the same team, and retained for 60 days.
```

Reply:
Suggestion would be to make the note more explicit, maybe use [admonitions](https://docs.saucelabs.com/contributing/style-guide/#admonitions)